task.go

```diff
@@ -61,6 +63,14 @@ func (t *task) process(conf *config) error {
 		}
 	}
 
+    if conf.flags&flagNoCmds == 0 {
```

Comment:
Combine the `if` check with the one directly below it, no need to perform the same test twice.